### PR TITLE
Updated README.md with build instructions

### PR DESCRIPTION
**Pull request by apham1**
Contains installation requirements as well as build instructions for running the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
 # FoodKeeper API
 API for USDA FSIS FoodKeeper data (<https://www.fsis.usda.gov/shared/data/EN/foodkeeper.json>)
 
-Rest API Endpoints:
+## Build Instructions
+### Requirements
+* Git Bash (<https://git-scm.com/downloads>)
```
Comment (kwurst) [inline README.md]: Remove. This is windows specific.

Maybe just "command line access"

File: README.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+
+### Application Download and Build
+```
+$ git clone https://github.com/LibreFoodPantry/FoodKeeper-API.git
+$ cd FoodKeeper-API\FoodKeeper-API
+
+$ gradle build
+$ gradle bootrun
+
+Native to localhost:8080/ and the application should be running.
```
Comment (kwurst) [inline README.md]: I'm not sure what you mean by "Native". Do you mean "Navigate"?

**Review by kwurst — CHANGES_REQUESTED** [review]
Please address these changes.

File: README.md (hunk 2 above)
Comment (apham1) [reply README.md]: Whoops, yeah it was supposed to be navigate.

**Comment by apham1** [comment]
I fixed the typo and also removed Git Bash as a part of the installation instructions in commit afec80b.

**Review by kwurst — APPROVED** [review]

**Pull request merged**